### ISSUE TITLE
Support for dotnet core LTS (2.1)?

### ISSUE THREAD

**Issue by rgl** [gabrielweyer/xunit-to-junit#1]
Is there a reason not to target dotnet core LTS (2.1)?

Installing on LTS, it gives the error:

```
C:> dotnet tool install --global dotnet-xunit-to-junit
error NU1202: Package dotnet-xunit-to-junit 1.0.0 is not compatible with netcoreapp2.1 (.NETCoreApp,Version=v2.1) / any. Package dotnet-xunit-to-junit 1.0.0 supports: netcoreapp2.2 (.NETCoreApp,Version=v2.2) / any
```

**Comment by gabrielweyer** [comment]
No reason, it was an oversight from my side 😄 I published `v1.0.1` which rectifies the issue by targeting `netcoreapp2.1`.

Sorry for the inconvenience.

**Comment by rgl** [comment]
It's working. Thank you!
